src/AbstractFixer.php

```diff
@@ -26,15 +31,25 @@
 abstract class AbstractFixer implements FixerInterface, DefinedFixerInterface
 {
     /**
+     * @var array<string, mixed>|null
+     */
+    protected $configuration;
+
+    /**
      * @var WhitespacesFixerConfig
      */
     protected $whitespacesConfig;
 
+    /**
+     * @var FixerConfigurationResolverInterface|null
+     */
+    protected $configurationDefinition;
```

Comment:
Not sure if we want to split `AbstractFixer`. If extracting configurable part, one could also want to extract whitespaceConfigurable part. but then we would end up in 2 new abstract classes, and then `header_comment` want to inherit both of them...
Please rethink that, if we really want to move that part away, we should do it fully, like moving `configure` into `AbstractConfigurableFixer`, this property and handling in constructor